gcloud/storage/blob.py

```diff
@@ -838,3 +900,21 @@ def __init__(self, bucket_name, object_name):
         self.query_params = {'name': object_name}
         self._bucket_name = bucket_name
         self._relative_path = ''
+
+
+def _set_encryption_headers(key, headers):
+    """Builds customer encyrption key headers
```

Comment:
would this be better as a "getter" for encryption headers and then the method using it does the `.update()` call?

Not super important, but figured I'd ask...